code/modules/reagents/reagents/medical.dm

```diff
@@ -1183,3 +1183,99 @@
 			continue
 		limb_to_unfix.fracture()
 		break
+
+
+/datum/reagent/medicine/research
+	name = "Research precursor" //nothing with this subtype should be added to vendors
+	taste_description = "bitterness"
+	reagent_state = LIQUID
+	taste_description = "bitterness"
+
+
+/datum/reagent/medicine/research/quietus
+	name = "Quietus"
+	description = "This is a latent poison, designed to quickly and painlessly kill you in the event that you become unable to fight. Never washes out on it's own, must be purged."
+	color = "#19C832"
+	custom_metabolism = 0
+	scannable = TRUE
+	taste_description = "Victory"
+
+/datum/reagent/medicine/research/quietus/on_mob_life(mob/living/L, metabolism)
+	switch(current_cycle)
+		if(1 to 59)
+			L.adjustStaminaLoss(1*effect_str)
+			if(prob(5))
+				to_chat(L, span_notice("You feel weakened by a poison."))
+		if(60)
+			to_chat(L, span_warning("You feel the poison settle into your body."))
+		if(61 to INFINITY)
+			if(L.stat == UNCONSCIOUS)
+				L.adjustOxyLoss(25*effect_str)
+				to_chat(L, span_userdanger("You fade into blackness as your lungs seize up!"))
+			if(prob(5))
+				L.adjustStaminaLoss(1*effect_str)
+	return ..()
+	
+/datum/reagent/medicine/research/quietus/on_mob_delete(mob/living/L, metabolism)
+	to_chat(L, span_danger("You convulse as your body violently rejects the suicide drug!"))
+	L.adjustToxLoss(30*effect_str)
+	
+	
+
+/datum/reagent/medicine/research/somolent
+	name = "Somolent"
+	description = "This is a highly potent regenerative drug, designed to heal critically injured personnel. Only functions on unconscious or sleeping people."
+	color = "#19C832"
+	custom_metabolism = REAGENTS_METABOLISM * 2.5 //0.5u/ticc
+	scannable = TRUE
+	taste_description = "naptime"
+
+/datum/reagent/medicine/research/somolent/on_mob_life(mob/living/L, metabolism)
+	if(current_cycle > 50)
+		return ..()
+	if(L.stat == UNCONSCIOUS)
+		L.heal_limb_damage(0.2*current_cycle*effect_str, 0.2*current_cycle*effect_str)
+		if(prob(20))
+			to_chat(L, span_notice("You feel as though you should be sleeping for the medicine to work."))
```

Comment:
but you're already sleeping..?

Reply:
Do you know of a message category that only functions while awake?